app/controllers/discourse_ai/sentiment/sentiment_controller.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+module DiscourseAi
+  module Sentiment
+    class SentimentController < ::Admin::StaffController
+      include Constants
+      requires_plugin ::DiscourseAi::PLUGIN_NAME
+      requires_login
```

Comment:
I think that we can drop `requires_login` as it is already handled by `::Admin::StaffController`